scripts/mkcloud

```diff
@@ -199,6 +201,21 @@ function error_exit()
     exit $exitcode
 } >&2
 
+function determine_host_mtu()
+{
+    local host_mtu
+
+    host_mtu=$(LC_ALL=C sort -n /sys/class/net/*/mtu | head -n 1)
+    # We may also have want_mtu_host so in that case we need to adjust
+    # it accordingly
+    if [[ $want_mtu_size -gt $host_mtu ]]; then
+        echo "Lowering vNIC MTU from $want_mtu_size to $host_mtu..."
+        want_mtu_size=$host_mtu
```

Comment:
wont this limit mtu to 1500 most of the time even when we want jumbo frames to be used?